app/models/grda_warehouse/warehouse_reports/time_homeless_for_exit.rb

```diff
@@ -73,13 +73,14 @@ def average_days
           )
         end
         # Layer on the first enrollment in PH if it is earlier
-        clients_with_housed_enrollments.order(e_t[:MoveInDate].asc).pluck(:id, e_t[:MoveInDate], p_t[:ProjectName]).each do |client_id, date, project_name|
+        clients_with_housed_enrollments.order(e_t[:MoveInDate].asc).pluck(:id, e_t[:MoveInDate], p_t[:ProjectName], p_t[:confidential], o_t[:confidential]).each do |client_id, date, project_name, p_confidential, o_confidential|
```

Comment:
any reason not to use `bool_or(p_t[:confidential], o_t[:confidential]).as('confidential')` here?

Reply:
no, just that I did this before I invented `bool_or`. I'll update.